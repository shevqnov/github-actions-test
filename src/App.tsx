import React from 'react';
import { Provider } from 'react-redux';
import Card from './components/Card';
import Hits from './components/Hits';
import SimpleForm from './SimpleForm';
import store from './store';

const someItems = [
  'item 1',
  'item 2',
  'item 3',
];

const Item = ({ name }: {name: string}) => (<li>{name}</li>);

const App = () => (
  <Provider store={store}>
    <ul>
      {someItems.map((item) => (
        <Item key={item} name={item} />
      ))}
      <Card text="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." title="Test title" image="no" />
      <SimpleForm validators={[() => true, (v) => typeof v === 'string']} />
      <Hits />
    </ul>
  </Provider>
);

export default App;
