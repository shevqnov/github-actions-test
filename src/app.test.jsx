import React from 'react';
import { mount } from 'enzyme';
import App from './App';

test('app should be rendered', () => {
  const wrapper = mount(<App />);
  expect(wrapper).toMatchSnapshot();
});
