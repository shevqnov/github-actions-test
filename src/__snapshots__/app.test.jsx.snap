// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should be rendered 1`] = `ReactWrapper {}`;
